internal/pkg/jsonnet/placeholder_test.go

```diff
@@ -0,0 +1,80 @@
+package jsonnet
+
+import (
+	"testing"
+
+	"github.com/google/go-jsonnet/ast"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConfigIdPlaceholder(t *testing.T) {
+	t.Parallel()
+	assert.Equal(t, `<<~~func:ConfigId:["my-config-id"]~~>>`, ConfigIdPlaceholder("my-config-id"))
+}
+
+func TestStripIdPlaceholder_NotFound(t *testing.T) {
+	t.Parallel()
+	assert.Equal(t, "", StripIdPlaceholder(""))
+	assert.Equal(t, "foo bar", StripIdPlaceholder("foo bar"))
+}
+
+func TestStripIdPlaceholder_ConfigId(t *testing.T) {
+	t.Parallel()
+	assert.Equal(t, "my-config-id", StripIdPlaceholder(ConfigIdPlaceholder("my-config-id")))
+}
+
+func TestStripIdPlaceholder_ConfigRowId(t *testing.T) {
+	t.Parallel()
+	assert.Equal(t, "my-config-row-id", StripIdPlaceholder(ConfigIdPlaceholder("my-config-row-id")))
+}
+
+func TestConfigRowIdPlaceholder(t *testing.T) {
+	t.Parallel()
+	t.Parallel()
+	assert.Equal(t, `<<~~func:ConfigRowId:["my-config-row-id"]~~>>`, ConfigRowIdPlaceholder("my-config-row-id"))
+}
+
+func TestReplaceFuncCallPlaceholders(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct{ input, expected string }{
+		{
+			input:    ``,
+			expected: `""`,
+		},
+		{
+			input:    `<<~~func:FuncName:["foo", "bar", 123]~~>>`,
+			expected: `FuncName("foo", "bar", 123)`,
+		},
+		{
+			input:    `in.c-keboola-ex-aws-s3-<<~~func:ConfigId:["om-default-bucket"]~~>>.table`,
+			expected: `"in.c-keboola-ex-aws-s3-" + ConfigId("om-default-bucket") + ".table"`,
```

Comment:
A tu mame testy, ktore to asi vysvetlia najlepsie.

`input` je nejaky string, ktory vieme ulozit aj ako Json.
`expected` je, ako sa ma tento `input` string ulozit do `JsonNet` suboru.

Reply:
Pouzitim vlastnej funkcie `ConfigId`, uz nemozu nastat pri kolizie, pri aplikovani sablony.
Ak by bol niekde string `om-default-bucket`, tak zostane zachovany.